packages/application-shell/src/components/redirect-to-project-create/redirect-to-project-create.tsx

```diff
@@ -27,19 +29,32 @@ export const RedirectToProjectCreate = () => {
   }
 
   return (
-    <Spacings.Stack>
-      <Text.Headline as="h3">Please create a project!</Text.Headline>
-      <Text.Body>
-        You are using the Merchant Center in development mode - it is not served
-        by a proxy (`env.json`). Moreover, you do not have any projects yet. As
```

Comment:
Since this message was referring to the `env.json`, I took the liberty to improve the UI a bit, as well as to adjust the information message.

Note that this message only appears in development, when the user has no projects.